src/test/java/com/google/songgame/TitleFormatterTest.java

```diff
@@ -0,0 +1,128 @@
+package com.google.songgame.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/*
+  https://music.youtube.com/playlist?list=RDCLAK5uy_kmPRjHDECIcuVwnKsx2Ng7fyNgFKWNJFs
+
+*/
+
+@RunWith(JUnit4.class)
+public final class TitleFormatterTest {
+
+  @Test
+  public void simpleCapitalization() {
+    TitleFormatter titleFormatter = new TitleFormatter();
+
+    String testString = "Marvins Room";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "marvins room");
+
+    testString = "BLACK PARADE";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "black parade");
+  }
+
+  @Test
+  public void removeOneGroup() {
+    TitleFormatter titleFormatter = new TitleFormatter();
+
+    String testString = "Sativa (ft. Swae Lee)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "sativa");
+
+    testString = "Get You (feat. Kali Uchis)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "get you");
+
+    testString = "Promises (feat. Namiko, Miyagi & Miyagi & Namiko)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "promises");
+
+    testString = "Tattoo (Remix)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "tattoo");
+  }
+
+  @Test
+  public void removeMultipleGroups() {
+    TitleFormatter titleFormatter = new TitleFormatter();
+
+    String testString = "Savage (Remix) (feat. Beyoncé)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "savage");
+
+    testString = "Life is Good (Official Music Video) (feat. Drake)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "life is good");
+  }
+
+  @Test
+  public void removeMultipleGroupsSquareAndParen() {
```

Comment:
nit: should be "removeMultipleGroupsSquareAndParenthesis". Normally you don't abbreviate words in Java identifiers.